src/runtime/description.ts

```diff
@@ -73,9 +73,9 @@ export class Description {
     const formatter = new (formatterClass)(this.particleDescriptions, this.storeDescById);
     const desc = formatter.getDescription(this.arcRecipes[this.arcRecipes.length - 1]);
     if (desc) {
-      return desc;
+      return [desc, false];
     }
-    return formatter._capitalizeAndPunctuate(this.arcRecipeName || Description.defaultDescription);
+    return [formatter._capitalizeAndPunctuate(this.arcRecipeName || Description.defaultDescription), true];
```

Comment:
1) the only reason we added `defaultDescription` in the first place was to avoid showing `undefined` suggestion in the suggestions box. so if we are not showing them, the defaultDescription is not needed either, i think. 
so i suggest simply returning undefined, if there is no description found (then filtering on non-undefined description in the plan consumer). this will keep the code simpler and function interface cleaner.
2) this is the only place where `this.arcRecipeName` is used, so if we choose not to display it as suggestion, let's just get rid of the data member. alternatively, you could check whether it is empty or not, and keep using the nonempty name as description. I do slightly prefer the former, but up to you.

Reply:
Thanks. I updated this PR with your comments. PTAL.